internal-import-file/import-document/src/reportimporter/report_parser.py

```diff
@@ -95,17 +95,33 @@ def parse(self, data: str) -> Dict[str, Dict]:
 
     def _parse_pdf(self, file_data: IO) -> Dict[str, Dict]:
         parse_info = {}
+        parsed_texts = []
+
+        def append_text_recursively(page_element):
+            for sub_element in page_element:
+                try:
+                    if isinstance(sub_element, LTTextContainer):
+                        parsed_texts.append(sub_element.get_text())
+                    else:
+                        append_text_recursively(sub_element)
+                except TypeError as error:
```

Comment:
You can test if an object is an Iterable with `isinstance(toto, Iterable)`

Reply:
I changed that, thanks!